src/rig_reconfigure.cpp

```diff
@@ -76,6 +77,28 @@ int main(int argc, char *argv[]) {
     glfwMakeContextCurrent(window);
     glfwSwapInterval(1); // Enable vsync
 
+    // load window icon
+    auto logoPath = std::filesystem::path(argv[0]).parent_path().append("resources/icon.png");
```

Comment:
Uh passt das immer? Überlebt das packaging und so?

Reply:
Ah ja und passt argv wenn man das via `ros2 run ... ` startet?

Reply:
Ich bin gerade wegen dem ros2 run auf das argv gegangen, da das in den install Ordner zeigt in den das Icon auch installiert wird. Aber ich stimme dir zu, dass das nicht immer klappt. Wenn man bspw. aus Clion heraus baut, wird das icon aus mir nicht bekannten Gründen nicht richtig installiert